modules/conversion-center/module.php

```diff
@@ -298,4 +338,32 @@ private function get_menu_args(): array {
 		];
 	}
 
+	private function add_links_pages_to_homepage_dropdowm( $output, $selected ) {
+		$args = array(
+			'posts_per_page' => 500,
+			'post_type' => self::CPT_LINKS_PAGES,
+		);
+
+		$posts = get_posts( $args );
+
+		try {
+			$dom = new DOMDocument();
```

Comment:
@louiswol94 is there any chance of DomDocument not being available?

Reply:
@nicoladj77 Let's split this dropdown into a separate task. I want to discuss it internally first.